src/Settings/components/theme.jsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { useState } from "react";
+import { Text, Appearance } from "react-native";
+import { ListItem, BottomSheet, Avatar, Icon } from "@rneui/themed";
+import STRINGS from "../../common/localization";
+import styles from "../styles";
+import { setTheme, THEMES, toggleNightMode } from "../../common/actions";
+import { useSelector } from "react-redux";
```

Comment:
As mentioned above too, move third party imports before the internal imports

Reply:
Fixed